src/util.cpp

```diff
@@ -729,14 +729,15 @@ CAboutDlg::CAboutDlg ( QWidget* parent ) : CBaseDlg ( parent )
                               "<p><b>" +
                               tr ( "Slovak" ) +
                               "</b></p>"
-                              "<p>Jose Riha (<a href=\"https://github.com/jose1711\">jose1711</a>)</p>" +
-                              "<p><b>" + tr ( "Simplified Chinese" ) +
+                              "<p>Jose Riha (<a href=\"https://github.com/jose1711\">jose1711</a>)</p>"
```

Comment:
This was almost correct. You need to add yourself without a "+" similar to the translators above.

Reply:
Thanks :) I would try again.